### PR TITLE
Fix location if section list is empty

### PR DESCRIPTION
**Pull request by arnaudon**

### CODE REVIEW

File: bluepyopt/ephys/locations.py [hunk 1]
```diff
@@ -521,10 +521,11 @@ def set_sec_index(self, icell=None):
                 for section in getattr(icell, self.seclist_name)
             ]
         )
-        if self.direction == 'radial':
-            self.sec_index = int(np.argmax(np.linalg.norm(points, axis=1)))
-        else:
-            self.sec_index = int(np.argmax(points.dot(self.direction)))
+        if len(points):
```
Comment (wvangeit) [inline bluepyopt/ephys/locations.py:524]: Can you be explicit here, prefer if statements with explicit equations

Comment (arnaudon) [reply bluepyopt/ephys/locations.py:524]: you mean adding >0?

File: bluepyopt/ephys/locations.py [hunk 2]
```diff
@@ -521,10 +521,11 @@ def set_sec_index(self, icell=None):
                 for section in getattr(icell, self.seclist_name)
             ]
         )
-        if self.direction == 'radial':
-            self.sec_index = int(np.argmax(np.linalg.norm(points, axis=1)))
-        else:
-            self.sec_index = int(np.argmax(points.dot(self.direction)))
+        if len(points) > 0:
```
[the remark this answers is not in the document]
Comment (arnaudon) [reply bluepyopt/ephys/locations.py:524]: yes, here: https://github.com/BlueBrain/BluePyOpt/blob/master/bluepyopt/ephys/locations.py#L432
as sec_index =None (default) value I didn't bother raising the same again

[the remark this answers is not in the document]
Comment (arnaudon) [reply bluepyopt/ephys/locations.py:524]: sure, better like that?

**Review by wvangeit — APPROVED** [review]

**Pull request merged**